### PR TITLE
docs: sidebar scrolling, responsive footer nav, re-org templates

### PR DESCRIPTION
**Pull request by hyu**
1. Added a lil animation for the sidebar nav items:

| Before | After |
| --- | --- |
|  |  | 

&nbsp; 
2. Adjusted the height of the sticky sidebar so you don't get the irritating thing where you can't "reach" the last few nav items.

| Before | After |
| --- | --- |
|   |  | 

&nbsp; 
3. Cleaned up the mobile nav footer a bit.

| Before | After |
| --- | --- |
| Screen Shot 2022-01-21 at 17 16 33 | Screen Shot 2022-01-21 at 17 18 28 | 

&nbsp; 
4. Renamed stuff in the `_layouts` folder. I wanted to make it more clear that two of these are wrappers, which the other layouts load via frontmatter. Not sure "layout_" is the best prefix, but at least it distinguishes the files when you look in the dir?

| Directory Structure | Usage |
| --- | --- |
| image | image |

### CODE REVIEW

File: src/_includes/header-docs.html [hunk 1]
```diff
@@ -2,18 +2,9 @@
 
 <header class="siteHeader2 siteHeader--docs">
   <div class="siteHeader-logoAndMenuButton">
-    {% if page.layout == 'landing' %}
```
Comment (hyu) [inline src/_includes/header-docs.html:5]: @lianmakesthings's [comment](https://github.com/tilt-dev/tilt.build/pull/1040#discussion_r789619500) about these silly obsolete lines inspired this grab-bag PR!

**Review by SurbhiSGupta — APPROVED** [review]

**Comment by SurbhiSGupta** [comment]
Thx Han.. looks great !

**Review by lizzthabet — APPROVED** [review]
looks great! ✨ the animations are a nice touch.

(optional nitpick) wdyt about `wrapper` as a prefix/suffix instead of `layout`? i don't find `layout` too descriptive from within the `layout` folder. 😅

**Comment by hyu** [comment]
> (optional nitpick) wdyt about `wrapper` as a prefix/suffix instead of `layout`? i don't find `layout` too descriptive from within the `layout` folder. 😅

Good point! I renamed to "wrapper-"

**Pull request merged**